### PR TITLE
[ubc-sunflower-genome] adding initial yaml descriptor for ubc-sungflower-genome

### PR DESCRIPTION
**Pull request by init-js**
*Description of changes:*

Adding the initial version of the UBC Sunflower Genome Dataset. It points to the S3 bucket containing the data, and points to the website for the project`s documentation. The DataAtWork section and publication records will be updated as the data is added.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/ubc-sunflower-genome.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+   - whole genome sequencing
+License: Public Domain
+Resources:
+  - Description: "UBC Sunflower Genome Data 1"
+    ARN: "arn:aws:s3:::ubc-sunflower-genome"
+    Region: "us-west-2"
+    Type: "S3 Bucket"
+DataAtWork:
+  Tutorials:
+    - Title:
```
Comment (chueatwork) [inline datasets/ubc-sunflower-genome.yaml:33]: Delete - Title: and following attributes if they're going to be empty!

File: datasets/ubc-sunflower-genome.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      URL:
+      AuthorName:
+      AuthorURL:
+  Tools & Applications:
+    - Title:
```
Comment (chueatwork) [inline datasets/ubc-sunflower-genome.yaml:38]: Same thing here

File: datasets/ubc-sunflower-genome.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      URL:
+      AuthorName:
+      AuthorURL:
+  Publications:
+    - Title:
```
Comment (chueatwork) [inline datasets/ubc-sunflower-genome.yaml:43]: And here

**Comment by init-js** [comment]
redoing. thank you.

**Pull request closed without merging**